packages/nextjs/app/admin/_components/GrantReview.tsx

```diff
@@ -48,7 +48,7 @@ export const GrantReview = ({ grant }: { grant: GrantDataWithBuilder }) => {
   });
   const sendTx = useTransactor();
 
-  const { handleReviewGrant: handleRejectGrant, isLoading } = useReviewGrant(grant);
+  const { handleReviewGrant, isLoading } = useReviewGrant(grant);
```

Comment:
Renaming this seemed a bit odd. Since the action is passed as an argument. nit pick, I know haha

cc @technophile-04

Reply:
Yup yup my bad, we were already nicely passing `PROPOSAL_STATUS.REJECTED` as param